### ISSUE TITLE
Flatpak depends on outdate and unsafe runtime

### ISSUE THREAD

**Issue by plumlis** [flathub/io.github.neil_morrison44.pocket-sync#83]
Info: runtime org.gnome.Platform branch 42 is end-of-life, with reason:
   The GNOME 42 runtime is no longer supported as of March 21, 2023. Please ask your application developer to migrate to a supported platform.
Info: applications using this runtime:
   io.github.neil_morrison44.pocket-sync

Info: runtime org.freedesktop.Platform.VAAPI.Intel branch 21.08 is end-of-life, with reason:
   org.freedesktop.Platform 21.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Info: applications using this extension:
io.github.neil_morrison44.pocket-sync

Info: runtime org.freedesktop.Platform.GL.default branch 21.08 is end-of-life, with reason:
   org.freedesktop.Platform 21.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Info: applications using this extension:
io.github.neil_morrison44.pocket-sync

**Comment by neil-morrison44** [comment]
https://github.com/neil-morrison44/pocket-sync/issues/107

TLDR is: can’t move runtime until Tauri 2 is released, which I’ll be moving to as soon as it’s out

**Comment by neil-morrison44** [comment]
Tauri 2's in beta now: https://beta.tauri.app/blog/tauri-2-0-0-beta/ hopefully it's not long before it gets release candidate'd and the documentation is written.

Should mean that all the EOF warnings go away, and a reasonable performance boost to the app on linux generally, with the loss of support of some older installations.

The migration is slightly more involved than I was thinking (almost everything in Tauri is a plugin in 2) & there's some things I want to move over to use the new proper "channels" API - but aside from that & any unforeseen issues I'll move it over fairly quickly

**Comment by hadess** [comment]
Fixed with pocket-sync 5.0.1
